content/testing/running-automated-tests.md

```diff
@@ -34,6 +34,23 @@ There are several options for running integration tests during the build. You ca
 * Android emulator
 * [AWS Device Farm](./aws) --- enables to run tests on physical Android and iOS devices. Requires an AWS account and additional setup.
 
+Devices available on the machine are:
+
+```sh
+> flutter emulators
+3 available emulators:
```

Comment:
While there are indeed 3 devices available, out of which two are Android emulators, it is advisible to use only Android emulator named `emulator` since this one has snapshot and starts faster. I'd alter the truth here a tiny bit and remove the `nc-emulator` reference entirely.

Reply:
Sure, sounds good!